ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -114,79 +119,86 @@ At least one queue type has been created, see [Create a queue type](#create-a-qu
+    + Specify the period in minutes in which the maximum number of actions are to be executed in the *Throttle period (minutes)* field. For example: Maximum number of actions = 1000, Throttle period = 60 means that a maximum of 1000 actions can be processed in one hour. 
+    Note that these numbers refer to a worker. If a worker is associated with multiple transitions, the number is divided among all the actions concerned. In addition, if more than one worker is configured for a queue type, there may be minimal deviations (one/two in a thousand) from the specified numbers due to parallel processing. 
 
-4. Unfold the engine room panel, disable the *Disable workflow* toggle in the *Actindo Work Flow Engine* box and fold the engine room panel again.
-    The workflow processes will start automatically.
+4. Click the [SAVE] button in the upper right corner.      
+    The changes have been saved. The *Edit queue type* view is closed. A confirmation message is displayed.
+
+     > [Info] An error notice is displayed if the number of booked vCores is not sufficient to create the queue type with the selected number of workers.
+
+    ![Maximum total workers](../../Assets/Screenshots/ActindoWorkFlow/QueueTypes/MaximumTotalWorkers.png "[Maximum total workers]")  
+
+5. Unfold the engine room panel, disable the *Disable workflow* toggle in the *Actindo Work Flow Engine* box and fold the engine room panel again.   
+    The workflow processes will start automatically after one minute. If you do not want to wait this time, see [Start workers](#start-workers).
 
     ![Enable workflow](../../Assets/Screenshots/ActindoWorkFlow/QueueTypes/EnableWorkflow.png "[Enable workflow]")
 
 
 
 ## Assign a queue type
 
-Assign a certain queue type to a transition, for instance to prioritize a certain action to be executed faster.
+Assign a certain queue type to a transition, for instance to prioritize a certain action to be executed faster, or to throttle an action, so that the system load is better distributed.
 
 #### Prerequisites
 
 At least one queue type has been created, see [Create a queue type](#create-a-queue-type).
 
 #### Procedure
 
-*Workflows > Workflows > Tab OVERVIEW > Select workflow > Select workflow version*
+*Process Orchestration > Workflows > Tab OVERVIEW > Select workflow > Select workflow version*
 
 ![Workflow editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
 
-1. Select the transition you want to assign a certain queue type to.      
-    The transition is highlighted and its settings are displayed in the settings side bar on the right.
+1. Select the transition to which you want to assign a certain queue type. If you want to throttle an action, in most cases it is recommended to assign the throttle queue type at one of the first transitions of the workflow.     
+    The transition is highlighted and its settings are displayed in the settings sidebar on the right.
```

Comment:
"Side bar" haben wir immer separat geschrieben. https://actindo.atlassian.net/wiki/spaces/DOC/pages/96862272/UI+Terminology

Im Wörterbuch finde ich eine andere Bedeutung, nicht Software-bezogen. Es gibt aber genug Webseiten, die darüber sprechen... Lass uns überlegen und entscheiden, damit wir es konsistent verwenden.
Das gilt auch für "ready for input" vs. locked/unlocked. Wir haben locked/unlocked Tausende mal...